extensions/dependencies/logic_proof.html

```diff
@@ -5,76 +5,3 @@
 <script src="{{cache_slug}}/extensions/interactions/LogicProof/static/js/generatedDefaultData.js"></script>
 <script src="{{cache_slug}}/extensions/interactions/LogicProof/static/js/generatedParser.js"></script>
 <script src="{{cache_slug}}/extensions/interactions/LogicProof/static/js/conversion.js"></script>
-<!-- TODO(sll): Find a way to move this to the HTML. -->
-<script type="text/ng-template" id="modals/logicProofHelp">
```

Comment:
@vojtechjelinek Where did this go? (Did you forget to add a file, or was this duplicating something else?)

(Make sure to manually test, btw.)

Reply:
This directive was moved to https://github.com/oppia/oppia/blob/develop/extensions/interactions/LogicProof/directives/logic_proof_help_modal_directive.html before.